errors.toml

```diff
@@ -1528,6 +1528,12 @@ description = ""
 workaround = "Please check if the previous worker is online."
 tags = ["internal", "high"]
 
+[error.DM-functional-34018]
+message = "inconsistent checkpoints between loader and target database"
+description = ""
+workaround = "Please check that you have not forgotten to use dmctl start-task --remove-meta=true. if you want to redo the task."
```

Comment:
If you want to redo the whole task, please check that you have not forgotten to add `-remove-meta` flag for `start-task` command.